nbs/010_engine.ipynb

```diff
@@ -24,8 +24,9 @@
    "outputs": [],
```

Comment:
<p>why do we have to do this manually? can we make this happen when the Engine is deleted? (via the <code>__del__</code> method?)  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/spannerlib/pull/162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>